deps/rabbit/priv/schema/rabbit.schema

```diff
@@ -2487,6 +2487,28 @@ end}.
 {mapping, "quorum_queue.property_equivalence.relaxed_checks_on_redeclaration", "rabbit.quorum_relaxed_checks_on_redeclaration", [
     {datatype, {enum, [true, false]}}]}.
 
+
+%%
+%% Queue membership evaluation
+%%
+
+{mapping, "quorum_queue_membership_evaluation.enabled", "rabbit.quorum_membership_changes_enabled", [
```

Comment:
Please move these under `quorum_queue.*`, for example,
`quorum_queue.continuous_membership_evaluation.enabled`, and so on.